app/src/main/java/com/jerboa/ui/components/common/AppBars.kt

```diff
@@ -127,15 +128,24 @@ fun BottomAppBarAll(
 
             NavigationBarItem(
                 icon = {
-                    Icon(
-                        imageVector = Icons.Outlined.List,
-                        contentDescription = "TODO",
-                    )
+                    if (screen == "communityList") {
+                        Icon(
+                            imageVector = Icons.Filled.List,
+                            tint = MaterialTheme.colorScheme.primary,
+                            contentDescription = "TODO",
```

Comment:
contentDescriptions are in another PR, but don't worry about them, as this will probably get merged before anyway.